src/backend/gporca/libgpopt/src/init.cpp

```diff
@@ -42,20 +42,27 @@ void
 gpopt_init()
 {
 	{
-		CAutoMemoryPool amp;
+		CAutoMemoryPool amp(CAutoMemoryPool::ElcStrict);
```

Comment:
Why has CAutoMemoryPool::ElcExc been replaced with CAutoMemoryPool::ElcStrict?

Reply:
> Why has CAutoMemoryPool::ElcExc been replaced with CAutoMemoryPool::ElcStrict?

To prevent the second scenario.